src/compressed_chunk.c

```diff
@@ -62,6 +62,9 @@ static void trimChunk(CompressedChunk *chunk) {
 
     if (excess > 1) {
         size_t newSize = chunk->size - excess + 1;
+        // align to 8 bytes (u_int64_t) otherwise we will have an heap overflow in gorilla.c because each write happens
+        // in 8 bytes blocks.
+        newSize += newSize % 8;
```

Comment:
```suggestion
        newSize += BIT - newSize % BIT;
```

Reply:
its better to align to have more space not less.